src/main/java/li/cil/tis3d/client/manual/segment/LinkSegment.java

```diff
@@ -95,7 +96,7 @@ private static void handleUrl(final String url) {
             final Object instance = desktop.getMethod("getDesktop").invoke(null);
             desktop.getMethod("browse", URI.class).invoke(instance, new URI(url));
         } catch (final Throwable t) {
-            MinecraftClient.getInstance().player.sendMessage(new LiteralText(t.toString()));
+            MinecraftClient.getInstance().player.sendSystemMessage(new LiteralText(t.toString()), Util.NIL_UUID);
```

Comment:
I have to admit to not having tested whether this change keeps the same behavior intact.

Reply:
Implementation looks to be the same, so this should be fine.